browser-test/src/applicant/applicant_prevent_duplicate_submission.test.ts

```diff
@@ -0,0 +1,124 @@
+import {expect, test} from '../support/civiform_fixtures'
+import {
+  enableFeatureFlag,
+  loginAsAdmin,
+  loginAsTestUser,
+  logout,
+  validateAccessibility,
+  validateScreenshot,
+  waitForPageJsLoad,
+} from '../support'
+
+test.describe('Prevent Duplicate Submission', () => {
+  const programName = 'Test Program Name'
+  const questionId = 'test-q'
+
+  test.beforeEach(async ({page, adminPrograms, adminQuestions}) => {
+    await test.step('Create program', async () => {
+      await loginAsAdmin(page)
+
+      await adminQuestions.addNumberQuestion({
+        questionName: questionId,
+        questionText: 'How many keyboards do you own?',
+      })
+
+      await adminPrograms.addProgram(programName)
+      await adminPrograms.editProgramBlockUsingSpec(programName, {
+        name: 'Screen 1',
+        description: 'First screen',
+        questions: [{name: questionId}],
+      })
+
+      await adminPrograms.publishProgram(programName)
+      await logout(page)
+    })
+  })
+
+  test(
+    'View Prevent Duplicate Submission page in old UI',
+    {tag: ['@northstar']},
```

Comment:
question (non-blocking): Should this be tagged `northstar` since it is the old ui?

Reply:
Good catch!